floris/tools/floris_interface.py

```diff
@@ -796,8 +784,7 @@ def get_farm_AEP(
         # Check if frequency vector sums to 1.0. If not, raise a warning
         if np.abs(np.sum(freq) - 1.0) > 0.001:
             self.logger.warning(
-                "WARNING: The frequency array provided to get_farm_AEP() "
-                "does not sum to 1.0."
+                "WARNING: The frequency array provided to get_farm_AEP() " "does not sum to 1.0."
```

Comment:
If this string is one line, you don't need the double-double quotes.

Reply:
Same thing, I think this came from ruff format. Now fixed.